build/tools/Automation/checkPipelineStatus.sh

```diff
@@ -0,0 +1,40 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+# This script will check the pipeline status' result
+# for a given a pipeline invocation id.
```

Comment:
Nit: very minor typo: `for a given a pipeline invocation id.` => `for a given pipeline invocation id.`

Reply:
Updated